test/helpers/data.go

```diff
@@ -52,6 +52,7 @@ func AppendRandomString(prefix string) string {
 // MakeK8sNamePrefix converts each chunk of non-alphanumeric character into a single dash
 // and also convert camelcase tokens into dash-delimited lowercase tokens.
 func MakeK8sNamePrefix(s string) string {
+	const dash = '-'
```

Comment:
there is already a const in the file called sep - https://github.com/knative/pkg/blob/master/test/helpers/data.go#L28